src/components/TeaserMyMagazine/TeaserMyMagazine.js

```diff
@@ -0,0 +1,258 @@
+                      ? `${document.meta.shortTitle.substring(0, 140).trim()} ${
+                          document.meta.shortTitle.length >= 140 ? '…' : ''
+                        }`
+                      : document.meta.title
+                  }
+                  credits={document.meta.credits}
+                />
+              ))}
+            </div>
+          )}
+        </div>
+      </section>
+    </div>
+  )
+}
+
+const styles = {
+  section: css({
+    margin: '0 auto',
+    maxWidth: 1300,
+    padding: '40px 15px 10px',
+    [mUp]: {
+      padding: '50px 15px 40px'
+    }
+  }),
+  row: css({
+    display: 'flex',
+    flexDirection: 'column',
+    [mUp]: {
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'flex-start'
+    }
+  }),
+  left: css({
+    marginBottom: 32,
+    [mUp]: {
+      flexGrow: 1,
+      marginRight: 16,
+      marginBottom: 0
+    }
+  }),
+  right: css({
+    [mUp]: {
+      flexGrow: 1,
+      marginLeft: 16
+    }
+  }),
+  center: css({
+    marginBottom: 0,
+    [mUp]: {
+      width: '50%',
+      marginBottom: 0
+    }
+  }),
+  tile: css({
+    padding: '16px 8px 12px 8px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    marginBottom: 16,
+    ':last-child': {
+      marginBottom: 30,
+      [mUp]: {
+        marginBottom: 40
+      }
+    },
+    [mUp]: {
+      padding: '12px 8px'
+    }
+  }),
+  tileHeadline: css({
+    textDecoration: 'none',
+    cursor: 'pointer',
+    wordWrap: 'break-word',
+    width: '100%',
+    ...serifBold17,
+    lineHeight: '18px',
+    [mUp]: {
+      ...serifBold19,
+      lineHeight: '21px'
+    }
+  })
+}
+
+const WrappedTeaserMyMagazine = props => (
+  <ColorContext.Provider value={colors.negative}>
+    <TeaserMyMagazine {...props} />
+  </ColorContext.Provider>
+)
+
+export default WrappedTeaserMyMagazine
+
+TeaserMyMagazine.propTypes = {
+  latestSubscribedArticles: PropTypes.array,
+  latestProgressOrBookmarkedArticles: PropTypes.array
+}
+
+WrappedTeaserMyMagazine.data = {
+  config: {
+    options: ({ first = 2 }) => ({
+      variables: {
+        first: +first
+      }
+    }),
+    props: ({ data }) => {
+      return {
+        data: {
+          loading: data.loading,
+          error: data.error,
+          latestSubscribedArticles: data.documents?.nodes.map(i => i.entity),
+          latestProgressOrBookmarkedArticles: data.me?.bookmarkAndProgress.filter(
+            i => i.document
+          )
+        }
+      }
+    }
+  },
+  query: `
+  query getMyMagazineDocuments {
+    documents: search(
+      filters: [
+          { key: "template", not: true, value: "section" }
+          { key: "template", not: true, value: "format" }
+          { key: "template", not: true, value: "front" }
+        ]
+        filter: { feed: true }
+        sort: { key: publishedAt, direction: DESC }
+        first: 2
+      ) {
```

Comment:
This query needs to change to get personalised documents. Something like:
```gql
{
  notifications(first: 2, filter: Document) {
    nodes {
      object {
        ... on Document {
          id
          meta {
            title
          }
        }
      }
    }
  }
}
```

Needs a backend change to allow to get only document notifications. I created a new filter for it in the backend: https://github.com/orbiting/backends/pull/514